### DISCUSSION TITLE
Code block in enumerated list

### DISCUSSION THREAD

**Discussion by superbarney** [TriliumNext/Trilium#2160] [Q&A]
I am struggling to find a way to add code block beneath an item in an enumerated list without it breaking the flow of the list, i.e. the numbering resets to 1. I hope I'm wrong but my search doesn't seem to have come up with any solutions. Has anyone succeeded in doing that before?

**Comment by zadam** [comment]
I think it's not possible ATM and lists can contain only "simple" elements. I noticed there's some work on "advanced lists" happening in CKEditor, maybe this will be part of that.
